### PR TITLE
Tidy some code

### PR DESCRIPTION
**Pull request by phil-davis**

### CODE REVIEW

File: src/gherkin/index.js [hunk 1]
```diff
@@ -205,8 +201,8 @@ class Step {
 }
 
 function verifyMatchParams(pattern) {
-  var reg = RegExp(/(\d+|"([^\"]*)"|'([^\']*)')/g) // eslint-disable-line no-useless-escape
-  var data = []
+  const reg = RegExp(/(\d+|"([^\"]*)"|'([^\']*)')/g) // eslint-disable-line no-useless-escape
+  const data = []
```
Comment (phil-davis) [inline src/gherkin/index.js:205]: Note: it feels strange, but seems to be the way of modern JS. An array can be declared `const` and `.push` can be used to add elements to the array. The constant thing is the pointer to the array, which is what is actually stored in the variable.

My IDE really wants me to code that way, even though it feels strange to me.

File: src/gherkin/stepDef.test.js [hunk 2]
```diff
@@ -138,7 +138,7 @@ describe('Checking stepDef generated patterns', () => {
         const stepDef = new StepDef(data.stepDef.token, data.stepDef.pattern, () => {
           return 0
         })
-        throw new Error('creating invalid stepdef should fail but it passed', stepDef)
+        throw new Error('creating invalid stepdef should fail but it passed')
```
Comment (phil-davis) [inline src/gherkin/stepDef.test.js:141]: `Error` only takes one parameter. 

If I put `+ stepDef` then maybe something will go wrong, or some really long sting representation of `stepDef` will be output.

File: src/helpers/config.js [hunk 3]
```diff
@@ -72,7 +72,7 @@ async function getConfigs() {
   const resp = await occHelper.runOcc(['config:list'])
   let stdOut = _.get(resp, 'ocs.data.stdOut')
   if (stdOut === undefined) {
-    throw new Error('stdOut notFound, Found:', resp)
+    throw new Error('stdOut notFound')
```
Comment (phil-davis) [inline src/helpers/config.js:75]: the value of `resp` might be anything. If I put `+ resp` then the string might be some really long representation of what?

**Review by kiranparajuli589 — APPROVED** [review]
lgtm :+1:

**Pull request merged**